.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -27,7 +27,7 @@ about: Tell us about a problem you are experiencing
 
 **Environment:**
 
-- Velero version (use `velero version`): 
```

Comment:
I think you switched the wrong one here (should be "Velero version (use `ark version`).:"